src/logic/messages/EventManager.cpp

```diff
@@ -105,6 +119,43 @@ void EventManager::processMessageQueue()
     if(m_EventQueue.empty())
         return;
 
+    bool print_queue = false;
```

Comment:
This code reveals a bug, if you test this with G2 and stand nearby Vatras he will spam thousands of messages.